war/src/main/scss/pages/_job.scss

```diff
@@ -0,0 +1,183 @@
+@use "../abstracts/mixins";
+
+#buildHistoryPage {
+  margin: 10px 0 10px 10px;
+
+  .jenkins-search {
+    margin-inline: -0.25rem;
+    margin-bottom: 5px;
+  }
+}
+
+.app-builds-container {
+  transition: opacity var(--standard-transition);
+
+  &__items {
+    margin-bottom: -0.5rem;
+  }
+
+  &__placeholder {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    padding: 3rem;
+    animation: fade-in-builds-placeholder var(--standard-transition);
+
+    @keyframes fade-in-builds-placeholder {
+      from {
+        opacity: 0;
+      }
+    }
+  }
+
+  &__heading {
+    display: flex;
+    font-size: 0.75rem;
+    color: var(--text-color-secondary);
+    margin-top: 10px;
+    font-weight: 450;
+    margin-bottom: 4px;
+  }
+
+  &__controls {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 15px;
+    margin: 0 -0.35rem;
+    margin-top: 1rem;
+    margin-bottom: -0.5rem;
+
+    .jenkins-button {
+      padding: 10px;
+
+      svg {
+        transition: translate var(--standard-transition);
+      }
+
+      &:first-of-type {
+        justify-content: start;
+
+        &:hover {
+          translate: -2px 0;
+
+          svg {
+            translate: -4px 0;
+          }
+        }
+      }
+
+      &:last-of-type {
+        justify-content: end;
+
+        &:hover {
+          translate: 2px 0;
+
+          svg {
+            translate: 4px 0;
+          }
+        }
+      }
+    }
+
+    .app-builds-container__button--disabled {
+      color: var(--text-color-secondary) !important;
+      opacity: 0.25;
+      pointer-events: none;
+    }
+  }
+
+  &--loading {
+    opacity: 0.4;
+    filter: blur(0.5px);
+  }
+}
+
+.app-builds-container__item {
+  @include mixins.item();
+
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  gap: 0.5rem 0.65rem;
+  padding: 0 0 0.25rem;
+  margin: 0 -0.5rem;
+  font-size: 0.8125rem !important;
+  min-height: 2rem;
+
+  &__icon {
+    display: inline-flex;
+    justify-content: center;
+    padding: 0 0 0 0.5rem;
+    margin-top: 0.385rem;
+
+    svg {
+      width: 1.25rem;
```

Comment:
This makes the status icon 20x20, before it was 16x16. Is this intentional?
The badges stay with 16x16

Reply:
I think so, think the status icon is the most important icon overall so doesn't hurt to be a touch larger.

Reply:
FWIW the icons in badges of course _can_ be bigger but really shouldn’t because they won’t align with the text height anymore. The latter is also the case with the build status icon being 20x20. I just found it odd that they became bigger (https://github.com/jenkinsci/badge-plugin/pull/151#issuecomment-2127234772)